css/area_footnotes.less

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file provides the design styles for the footnotes.
+ *
+ */
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes section  */
+.dokuwiki div.footnotes {
+  div.fn {
+    margin-bottom: .2rem;
+    display: table;//
+    vertical-align: top;//
+
+    sup, //
+    .content {
+      display: table-cell;
+    }
+
+    sup { //
+      vertical-align: top;
+
+      // footnote anchor
+      a.fn_bot {
+        font-size: .86em;
+        padding-right: .2em;
+      }
+    }
+
+    .content { //
+      vertical-align: top;
+      line-height: 135%;
+    }
+
+    > sup {
+
+    }
+
+    // footnote text
+    div.content {
+      line-height: 135%;
+    }
+  }
+}
+
+
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes in content  */
+sup,
+.main-content sup,
+.main-content > div > p sup,
+.main-content > div > ul > li .li sup
```

Comment:
Why do we need so many selectors here?

Reply:
unfortunately these nested selectors are necessary 
(overwrite "fonts-size: inherit" of link elements in main-content without changing default font-size of sup)